include/ensmallen_bits/de/de_impl.hpp

```diff
@@ -0,0 +1,125 @@
+#ifndef ENSMALLEN_DE_DE_IMPL_HPP
+#define ENSMALLEN_DE_DE_IMPL_HPP
+
+#include "de.hpp"
+#include <random>
+#include <tuple>
+#include <iostream>
+
+namespace ens {
+
+	inline DE::DE( const size_t populationSize ,
```

Comment:
Can you adapt the style from the existing codebase? For more information see https://github.com/mlpack/mlpack/wiki/DesignGuidelines.

Reply:
I had not seen this, i will make the necessary fixes